Tasks/VsTest/vstest.ts

```diff
@@ -34,6 +34,8 @@ try {
     var configuration: string = tl.getInput('configuration');
     var publishRunAttachments: string = tl.getInput('publishRunAttachments');
     var runInParallel: boolean = tl.getBoolInput('runInParallel');
+    var tiaEnabled = tl.getVariable('tia.enabled');
+    var tiaResponseFile = tl.getVariable('tia.responsefile');
```

Comment:
do we want this path to be given by user? i think we shuld generate this ourselves instead.
also the took location (testselector) can be a variable today